src/main/scala/nl/knaw/dans/lib/logging/servlet/package.scala

```diff
@@ -47,6 +47,10 @@ import org.scalatra.ActionResult
  *    import org.scalatra.{ Ok, ScalatraServlet }
  *
  *    class ExampleServlet extends ScalatraServlet with ServletLogger with DebugEnhancedLogging {
+ *
+ *      // I'd like to see a mandatory choice between a MaskedLogFormatter, a PlainLogFormatter or a CustomLogFormatter.
```

Comment:
@jo-pol do I understand you correctly in saying that you want something like:

```scala
class ExampleServlet extends ScalatraServlet with ServletLogger with PlainServletLogFormatter with DebugEnhancedLogging
```

wouldn't that be adding extra complexity to the whole thing?